pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -58,6 +74,29 @@ pub mod pallet {
 	#[pallet::getter(fn organizer)]
 	pub type Organizer<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
 
+	#[pallet::storage]
+	pub type MintAvailable<T: Config> = StorageValue<_, bool, ValueQuery>;
+
+	#[pallet::type_value]
+	fn DefaultMintFee<T: Config>() -> BalanceOf<T> {
+		(1_000_000_000_000_u64 * 55 / 100).into()
```

Comment:
Use `sp_runtime::traits::UniqueSaturatedInto` trait here:
```suggestion
	pub fn DefaultMintFee<T: Config>() -> BalanceOf<T> {
		(1_000_000_000_000_u64 * 55 / 100).unique_saturated_into()
```

Reply:
`Currency::Balance` type is bound with [AtLeast32BitUnsigned trait](https://github.com/paritytech/substrate/blob/563f4820d8f36d256ada7ea3fef46b2e94c4cd5a/frame/support/src/traits/tokens/misc.rs#L167-L170) which implements `UniqueSaturatedInto` for `u64`